engine/src/main/java/org/terasology/input/InputSystem.java

```diff
@@ -64,6 +65,7 @@
  * In addition to raw keyboard and mouse input, the system handles Bind Buttons and Bind Axis, which can be mapped
  * to one or more inputs.
  */
+@RegisterSystem
```

Comment:
I wonder if this should be set to Client? It'll otherwise default to Always which means it would also run on a headless server, which seems silly for input handling.

Which makes me wonder what it was doing _without_ the annotation in the first place 🤔 I'm tempted to put a couple fingers in my ears and walk away slowly going "lalalalala" pretending to never having seen this ... 😁

Reply:
I got the impression that we should double-check all `@RegisterSystem` usages anyways 🙈 Having it `ALWAYS` register wouldn't hurt that much, would it?

Reply:
Probably not, but the class _does_ have an `update` that'll run every tick, pointlessly, since servers don't process input. Likely it is minor and probably even has been an issue for ages. So not likely much to worry about - we should hunt whichever performance issues actually manifest themselves rather than theoretical ones :-)

(and yeah we should totally check all the systems - if no sooner then maybe when we re-integrate Gestalt, as any detailed effort until then might end up wasted if everything changes too much anyway ...)